x-pack/plugins/session_view/common/constants.ts

```diff
@@ -13,7 +13,7 @@ export const PROCESS_EVENTS_INDEX = 'logs-endpoint.events.process-*';
 export const PREVIEW_ALERTS_INDEX = '.preview.alerts-security.alerts-default';
 export const ENTRY_SESSION_ENTITY_ID_PROPERTY = 'process.entry_leader.entity_id';
 export const ALERT_UUID_PROPERTY = 'kibana.alert.uuid';
-export const KIBANA_DATE_FORMAT = 'MMM DD, YYYY @ hh:mm:ss.SSS';
+export const KIBANA_DATE_FORMAT = 'MMM DD, YYYY @ HH:mm:ss.SSS';
```

Comment:
I think eventually this should come from the user configurable global kibana setting, example can be found here https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/lib/kibana/hooks.ts#L27

Reply:
good point, I will add that for us to improve later, thanks!